### ISSUE TITLE
feature request: "Typing mode to Visual mode" hotkey

### ISSUE THREAD

**Issue by pvutov** [alacritty/alacritty#6810]
**Description**
I want to press a hotkey and go from typing to Vi selection mode. So the same effect as pressing `ctrl+shift+space` followed by  `V`, but in one key.
Maybe this is already supported and I just don't understand the config, but I tried this:

```
# Second command is not executed
- { key: V, mods: Super, mode: ~Search, action: ToggleViMode }
- { key: V, mods: Super, mode: ~Search|Vi, action: ToggleNormalSelection }
```

The other approaches I tried are:
```
# Complains that ToggleNormalSelection is only available in Vi mode
- { key: V, mods: Super, mode: ~Search, action: ToggleNormalSelection }
```
and 
```
# Can only have one action in the action field
- { key: V, mods: Super, mode: ~Search, action: ToggleViMode,ToggleNormalSelection }
```

**Use case**
I don't know how others use vi mode. At least for me, this is a big use case, because I mostly use vi mode to copy text, which currently seems to require highlighting (since we are limited to single keys and not things like `yy` etc). 90% of the time I am doing `ToggleViMode` immediately followed by `ToggleNormalSelection`.

**Comment by kchibisov** [comment]
Yeah, I think you can't combine bindings like that across the mods. Maybe you could do something like

```
- { key: V, mods: Super, mode: ~Search, action: ToggleViMode }
- { key: V, mods: Super, mode: ~Search, action: ToggleNormalSelection }
```

But I'm not sure.

**Comment by pvutov** [comment]
That does not work, because putting `ToggleNormalSelection` in the same line as `mode: ~Search`  triggers an error (second example in OP).

```
Config error: `ToggleNormalSelection` is only available in vi mode, try adding `mode: Vi`; ignoring binding
```

**Comment by chrisduerr** [comment]
I think we should definitely figure out how to combine keybindings across mode switches. It's something that has frequently come up in the past and I feel like it shouldn't be that hard to fix.

**Comment by kchibisov** [comment]
I think it's done, but we forgot to close the issue. See 586f982a260a1864f61f9aff6a861d3231bf3301

**Comment by pvutov** [comment]
I am still getting the `ToggleNormalSelection is only available in Vi mode` warning, guess the commit isn't in 0.12.1? Gonna try again when 0.13 lands in arch, thanks for implementing it.

**Comment by kchibisov** [comment]
yeah, it's not released, will be in 0.13.0.
